docs/guides/update-guide/introduction.md

```diff
@@ -3,33 +3,37 @@ id: introduction
 title: Introduction
 ---
 
-These documents guide you through the process of updating your Camunda Cloud
-application or server installation from one Camunda Cloud version to the other.
+These documents guide you through the process of updating your Camunda Platform 8
+application or server installation from one Camunda Platform 8 version to the other.
+
+:::note
+Versions prior to Camunda Platform 8 are listed below and identified as Camunda Cloud versions.
+:::
 
 There is a dedicated update guide for each version:
 
-### [1.2 to 1.3](../120-to-130)
+### [Camunda Cloud 1.2 to 1.3](../120-to-130)
```

Comment:
what about the guide from Camunda Cloud 1.3 to Camunda Platform 8.0? is this readyalready?

Reply:
@menski will you PR this or delegate?

Reply:
it's on my list